### PR TITLE
Change access modifiers

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
When supporting custom LSP extensions, the developer needs more accessibility for the client components. This PR updates access modifiers of few such components.

### CODE REVIEW

**Comment by nixel2007** [comment]
No getters?

File: src/main/java/org/wso2/lsp4intellij/client/languageserver/requestmanager/DefaultRequestManager.java [hunk 1]
```diff
@@ -87,13 +87,14 @@
 public class DefaultRequestManager implements RequestManager {
 
     private Logger LOG = Logger.getInstance(DefaultRequestManager.class);
+
+    public LanguageServerWrapper wrapper;
+    public LanguageServer server;
+    public LanguageClient client;
+    public ServerCapabilities serverCapabilities;
```
Comment (rasika) [inline src/main/java/org/wso2/lsp4intellij/client/languageserver/requestmanager/DefaultRequestManager.java]: If there's no special reason; shall we make these fields `private` and add some **getters** for these. I believe these values are being passed through the **constructor**. Allowing them `public` will make it harder to debug and test the code.

File: src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java [hunk 2]
```diff
@@ -172,10 +172,10 @@
     public Editor editor;
     public LanguageServerWrapper wrapper;
     public List<String> completionTriggers;
-    protected Project project;
-    private RequestManager requestManager;
+    public Project project;
+    public RequestManager requestManager;
+    public TextDocumentIdentifier identifier;
```
Comment (rasika) [inline src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: Same goes here. Keep them as **private** and add **getters**.

**Comment by NipunaRanasinghe** [comment]
> Not getters?

@nixel2007 Just following YAGNI :D Will add getters anyway

File: src/main/java/org/wso2/lsp4intellij/client/languageserver/requestmanager/DefaultRequestManager.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply src/main/java/org/wso2/lsp4intellij/client/languageserver/requestmanager/DefaultRequestManager.java]: Added :)

File: src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java (hunk 2 above)
Comment (NipunaRanasinghe) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: Added.

**Pull request merged**